### PR TITLE
Notify

### PR DESCRIPTION
**Pull request by joredudiaz**
# Summary

-  notify to an user when a rent is expiration

# Trello card
https://trello.com/c/1EdctguQ

### CODE REVIEW

File: app/mailers/notify_mailer.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+    I18n.with_locale(set_locale) do
+      mail(to: @current_rent.user.email, subject: I18n.t(:subject_notify_email))
+    end
+  end
+
+  private
+
+  def set_locale
+    @current_rent.user.try(:locale) || I18n.locale_default
```
Comment (holywyvern) [inline app/mailers/notify_mailer.rb]: Avoid `Object#try` and use `object&.locale`

File: app/workers/notify_end_rent_worker.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+require 'sidekiq-scheduler'
+
+class NotifyEndRentWorker
+  include Sidekiq::Worker
+
+  def perform
+    logger.info 'Verifying rents...'
+    Rent.where(end_date: Time.zone.now).pluck(:id).each do |rent_id|
```
Comment (holywyvern) [inline app/workers/notify_end_rent_worker.rb]: This only works if this scheduler runs every second, it's better if you add a "notified" field to rent and check when end_date is older than Time.zone.now and it's not notified.

**Review by holywyvern — CHANGES_REQUESTED** [review]

File: app/mailers/notify_mailer.rb (hunk 1 above)
Comment (joredudiaz) [reply app/mailers/notify_mailer.rb]: upss....done!

File: app/workers/notify_end_rent_worker.rb (hunk 2 above)
Comment (joredudiaz) [reply app/workers/notify_end_rent_worker.rb]: right! done!

**Pull request merged**